README.md

```diff
@@ -177,8 +183,8 @@ less: {
     options: {
       paths: ["assets/css"],
       plugins: [
-        new (require('less-plugin-autoprefix'))({browsers: ["last 2 versions"]}),
-        new (require('less-plugin-clean-css'))(cleanCssOptions)
+        new require('less-plugin-autoprefix')({browsers: ["last 2 versions"]}),
```

Comment:
Can we merge it?

If this will cause delays lets just pull it out and get the functionality in and let the readme be compiled directly.